forc/src/cli/commands/plugins.rs

```diff
@@ -11,33 +11,68 @@ pub struct Command {
     /// Prints the absolute path to each discovered plugin.
     #[clap(long = "paths", short = 'p')]
     print_full_path: bool,
+    /// Prints the long description associated with each listed plugin
+    #[clap(long = "describe", short = 'd')]
+    describe: bool,
 }
 
 pub(crate) fn exec(command: PluginsCommand) -> Result<()> {
-    let PluginsCommand { print_full_path } = command;
+    let PluginsCommand {
+        print_full_path,
+        describe,
+    } = command;
 
+    println!("Installed Plugins:");
     for path in crate::cli::plugin::find_all() {
-        print_plugin(path, print_full_path);
+        print_plugin(path, print_full_path, describe);
     }
     Ok(())
 }
 
+fn parse_description_for_plugin(plugin: &str) -> String {
+    use std::process::Command;
+    let proc = Command::new(plugin)
```

Comment:
I'm not sure if there's a more idiomatic way to grab plugin descriptions

Reply:
Not that i'm aware of currently. There is a `print_help` method in `Clap` but I don't think that helps us here. https://docs.rs/clap/latest/clap/struct.App.html#method.print_help